mews-operations/reservations.md

```diff
@@ -263,10 +267,18 @@ The third `reservation` definition shows the partial cancellation - cancelling t
 | `title` | `string` [Title](#title) | optional | Customer title, e.g. "Mister" |
 | `telephone` | `string` | optional | Telephone. |
 | `loyaltyCode` | `string` | optional | Loyalty code of the customer. Processed only within main customer. |
+| `loyaltyInfo` | [`LoyaltyInfo`](#loyalty-info) object | optional | Loyalty information of the customer. Processed only within main customer or [`Availability block`](../channel-manager-operations/availabilityBlock.md#availability-block) booker. |
```

Comment:
I'm not sure what is meant by "Processed only within main customer or Availability Block booker", is it possible to put this into different words?

Reply:
it is copied from above, but sure, we can change it.

it is supposed so say, that it is taken/send only for the main customer/block booker. for other `customer` profiles in that message we ignore the value (even if provided)

Reply:
Ah, thanks for the explanation @ondrahermanek. In that case, I would suggest the following very small change:

**Loyalty code:**
Loyalty code of the customer. Processed only for the main customer.

**Loyalty info:**
Loyalty information of the customer. Processed only for the main customer or Availability block booker.